modules/ROOT/pages/prereq-env.adoc

```diff
@@ -0,0 +1,39 @@
+= Environment Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Adhere to the follow recommendations and limitations when setting up your environment and deploying apps:
+
+* Environment:
+
+** Production: 6 Nodes of 8 cores and 32GB 
+
+* Sizing
+
+** ARM:
+
+*** Applications per Environment 
+
+**** Less than 200 
+
+***** Maximum: 500 Applications in a single environment 
```

Comment:
Should Applications be lowercase (and maybe "apps"?) 

Global: look at capitalization and apply Kim's rules  :)

Reply:
Got it - thanks!